SerialPrograms/Source/PokemonSV/Programs/FastCodeEntry/PokemonSV_CodeEntry.cpp

```diff
@@ -72,19 +72,22 @@ const char* normalize_code(std::string& normalized_code, const std::string& code
 
     switch (normalized_code.size()){

```

Comment:
couldn't you return here if override_mode ?

Reply:
added return if override before the other checks